packages/config/src/projects/intmax/intmax.ts

```diff
@@ -21,14 +36,160 @@ export const intmax: ScalingProject = underReviewL2({
       ],
```

Comment:
They have nice docs now: https://intmax-wallet.gitbook.io/intmax-developers-hub. Also this: https://docs.network.intmax.io